react-native/react/constants/update.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+
+import type {TypedAction} from '../constants/types/flux'
+import type {Asset, UpdateType} from '../constants/types/flow-types'
+
+export const showUpdatePrompt = 'update:showUpdatePrompt'
+export type ShowUpdateAction = TypedAction<'update:showUpdatePrompt', showUpdatePromptPayload, UpdateError>
+
+export const registerUpdateListener = 'update:registerUpdateListener'
+export type RegisterUpdateListenerAction = TypedAction<'update:registerUpdateListener', any, any>
```

Comment:
The second parameter here should be `{started: boolean}` since that's the type of the payload for this action.